Identity_Application/Services/Authentication/PasswordHasherService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Identity_Application.Interfaces.Authentication;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Identity_Application.Services.Authentication;
+
+public class PasswordHasherService : IPasswordHasherService
+{
+    public string ComputeHash(string password, string salt, string pepper, int iteration)
```

Comment:
we are only interested in checking out the password (because everything else is generated automatically), and it is checked at higher levels. Do we need to do that check twice ?

Reply:
fixed